tmt/base.py

```diff
@@ -3331,6 +3345,13 @@ def show_runner(self, logger: tmt.log.Logger) -> None:
         for _, key_formatted, value_formatted in self.runner.facts.format():
             log(key_formatted, value_formatted, shift=1)
 
+    def prepare(self, tree: Tree) -> None:
+        """ Prepare the run for the try command """
+        self.tree = tree
+        self._workdir_load(self._workdir_path)
+        self.config.last_run = self.workdir
+        self.info(str(self.workdir), color='magenta')
```

Comment:
I think `str` is no longer needed, `info()` should be able to handle `Path` out of the box.

Reply:
Right, removed in 93c1eec1.

Reply:
Hm, interesting, even with `assert self.workdir is not None` (tried locally), `mypy` is still unhappy:

    tmt/base.py:3357: error: Argument 1 to "info" of "Common" has incompatible type "Path"; expected "str"  [arg-type]

Any idea why? I see `Path` in the list of `LoggableValue`s...

Reply:
Aha, that's actually right - `Path` is acceptable as the second parameter, `value`, not the first one, `key`. `key` must be a string, therefore `str()` needs to stay.

Reply:
Right, reverted in 0242d7fd.